pkgs/development/python-modules/warble/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, gcc49
+, cython
+, libcxx
+, boost
+, bluez
+}:
+
+buildPythonPackage rec {
+  pname = "warble";
+  version = "1.2.9";
+  format = "setuptools";
+
+  enableParallelBuilding = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-oezcRD1AddWmDYDxueE0EwK0+UN/EZ5GQxwkdCz4xoY=";
+  };
+
+  nativeBuildInputs = [
+    gcc49
```

Comment:
We can't use a gcc from the last years?

Reply:
Thanks for the review, I just tested it with `gcc` and `gcc13`, both of them compiles just fine.

The reason why I put `gcc49` in the first place was because https://github.com/NixOS/nixpkgs/pull/238630 (a package depends on `warble`) does not compile with `gcc` and `gcc13` (error with `nlohmann_json`), and I just copy-pasta-ed everything from there.